### ISSUE TITLE
[WD recon] "Match other cells with same content" not precise enough

### ISSUE THREAD

**Issue by VladimirAlexiev** [OpenRefine/OpenRefine#2332]
I have a sheet like this:
```
oblast  city     museum
Varna   Varna    History museum
Varna   Nesebar  History museum
```

I've reconciled columns 1 and 2, now trying to reconcile column 3; I have specified to use "oblast" and "city" as extra props "located in the administrative entity".
- the candidates for column 3 don't reflect the first two cols at all. But let's say I could understand this because of the ambiguity between using col 1 or 2.
- After doing "search for match" and selecting the correct museum, by default it does "Match other cells with same content" and takes into account the text of *column 3 only*. This is silly: column 3 on itself doesn't mean much, you have to take the other columns in account to disambiguate. I have reconciled them, so why doesn't it take them into account?

Please add a first option "Match other rows with same content" that takes into account all cols used in reconciling the current column. Make it the default.

**Comment by VladimirAlexiev** [comment]
I have not 2 but maybe 10 rows all saying "History museum".
I swear that just after reporting this bug, I screwed myself over 3 times by searching for the appropriate museum, then clicking which screwed up the other *already reconciled* rows.

That's how misleading the default option "Match other cells with same content" is !!!

**Comment by wetneb** [comment]
Completeley legitimate request - it would make a lot of sense.
The ReconConfig object in the column metadata can be used to retrieve the reconciliation settings so an operation could easily take into the other columns used during reconciliation for this matching.

**Comment by thadguidry** [comment]
I think it should be renamed to "Match other cells with same text"

![image](https://user-images.githubusercontent.com/986438/75352128-da47b200-586e-11ea-9765-e1cfe2ee7a8c.png)

**Comment by VladimirAlexiev** [comment]
- Proposed in #2316 to make"Match this cell **only**" the first and default choice.
- Agree about renaming to "Match other cells with same **text**".
- Then my proposed option "Match other **rows with same content**" will stand out even better

**Comment by thadguidry** [comment]
@VladimirAlexiev The fuller way to say it would be "Match other rows **in this column** with same text" otherwise it might be unclear for beginners of OpenRefine.

**Comment by wetneb** [comment]
I would potentially remove the "Match other cells with same text" from the UI and only keep "this cell only" and "rows with same content", to keep the dialog simple. It would become clear that "rows with the same content" is an improvement of "same text".

**Comment by kushthedude** [comment]
@wetneb @VladimirAlexiev Can I go ahead and make the changes requested in here?

**Comment by wetneb** [comment]
Please do! This is slightly more involved than other issues you have been tackling so far, so it's a good next step.

**Comment by VladimirAlexiev** [comment]
@wetneb I agree, I can't think of a useful case where extra rows used in recon should be disregarded when proliferating the match. 

@thadguidry I think you overdid the message a bit, now I can't understand it :)
